### DISCUSSION TITLE
I'd like to help with Langchain documentation

### DISCUSSION THREAD

**Discussion by funkytaco** [FlowiseAI/Flowise#143] [General]
Back in the 90's we used to say RTFM in IRC, and it appears that changed to check the youtube video or stack overflow. :)

I'd like to help with Langchain documentation if the team needs any assistance. I was thinking maybe Markdown for documentation, which can be converted into other formats, such as PDF, which can then be fed to an LLM. :) 

Thoughts?

**Comment by HenryHengZJ** [comment]
hey thanks for offering the help!

There is docs for langchainjs already - https://js.langchain.com/docs/getting-started/guide-llm

As for flowise docs, its work in progress! will release that soon and welcome any contributions, just need to get it started first!

**Comment by funkytaco** [comment]
Okay, thanks.

**Reply by chungyau97** [reply]
Hi @funkytaco,

Here is the [Flowise Docs GitHub](https://github.com/flowiseai/flowisedocs) and [Flowise Docs Website](https://docs.flowiseai.com/)
